RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelClusterThresholds.h

```diff
@@ -0,0 +1,13 @@
+#ifndef RecoLocalTracker_SiPixelClusterizer_plugins_SiPixelClusterThresholds_h
+#define RecoLocalTracker_SiPixelClusterizer_plugins_SiPixelClusterThresholds_h
+
+struct SiPixelClusterThresholds {
+  SiPixelClusterThresholds() = default;
+  inline constexpr int32_t getThresholdForLayerOnCondition(bool isLayer1) const noexcept {
+    return isLayer1 ? kDefaultPhase1_layer1 : kDefaultPhase1_otherLayers;
+  }
+  const int32_t kDefaultPhase1_layer1 = 2000;
```

Comment:
this is rather confusing now; only `2000` is properly a default for phase1.
The actual configured values which are not `2000` are not a default anymore.

The reason I proposed the constants was to use the same value in SiPixelDigisClustersFromSoA.cc and SiPixelRawToClusterCUDA.cc fillDescriptions so that we have only one place to define constants which are supposed to have the same value.
However, the recent update introduced now the third repeated definition and made the naming somewhat more confusing.

The original names `layer1` and `otherLayers` was more appropriate for general use context.